docs/src/main/resources/content/config.adoc

```diff
@@ -0,0 +1,38 @@
+Version ${project.version}. Connexta
```

Comment:
❔ Do we need the Connexta here? Should we switch to Octo? Not sure how we are handling that with portions that have existed for years vs new docs/code.

Reply:
> @ricklarsen Just wanted you to know that the master branch of this repo isn't used by any projects I am aware of. The 0.2.x and 0.3.x branches are though. The master branch started a transition of the service but was moved to a different repo part way through.
> 
> The documentation that was created for this repo is mostly only valid for the 0.2.x version.

thanks! I'll update for that

Reply:
> ❔ Do we need the Connexta here? Should we switch to Octo? Not sure how we are handling that with portions that have existed for years vs new docs/code.

That's a good question. I imagine we'd update it, but I'll be happy to revert if I'm wrong